modules/web/js/ballerina/views/ballerina-file-editor.js

```diff
@@ -767,6 +775,13 @@ define(['lodash', 'jquery', 'log', './ballerina-view', './service-definition-vie
             // adding current package to the tool palette with functions, connectors, actions etc. of the current package
             this.addCurrentPackageToToolPalette();
             this._model.accept(this);
+
+            // adding declared import packages to tool palette
+            _.forEach(this._model.getImportDeclarations(), function (importDeclaration) {
+                var package = BallerinaEnvironment.searchPackage(importDeclaration.getPackageName());
```

Comment:
This view is now rendered as a react component. So code is refactored. The relevant variable is [renamed to `imports`](https://github.com/ballerinalang/composer/blob/master/modules/web/js/ballerina/tool-palette/tool-palette-view.jsx#L441).